app/components/candidate_interface/decoupled_references_review_component.rb

```diff
@@ -106,38 +106,36 @@ def feedback_status_text(reference)
     end
 
     def feedback_status_content(reference)
-      if reference.not_requested_yet?
-        tag.p(t('application_form.referees.info.not_requested_yet'), class: 'govuk-body govuk-!-margin-top-2')
-      elsif reference.feedback_refused?
-        tag.p(t('application_form.referees.info.declined'), class: 'govuk-body govuk-!-margin-top-2')
-      elsif reference.cancelled_at_end_of_cycle?
-        tag.p(t('application_form.referees.info.cancelled_at_end_of_cycle'), class: 'govuk-body govuk-!-margin-top-2')
-      elsif reference.cancelled?
-        tag.p(t('application_form.referees.info.cancelled'), class: 'govuk-body govuk-!-margin-top-2')
-      elsif reference.feedback_overdue?
-        tag.p(t('application_form.referees.info.feedback_overdue'), class: 'govuk-body govuk-!-margin-top-2')
-      elsif reference.feedback_requested? && reference.requested_at > Time.zone.now - 5.days
-        tag.p(t('application_form.referees.info.awaiting_reference_sent_less_than_5_days_ago'), class: 'govuk-body govuk-!-margin-top-2')
-      elsif reference.feedback_requested?
-        tag.p(t('application_form.referees.info.awaiting_reference_sent_more_than_5_days_ago'), class: 'govuk-body govuk-!-margin-top-2')
+      text =
+        if reference.feedback_refused?
+          t('application_form.referees.info.declined', referee_name: reference.name)
+        elsif reference.cancelled_at_end_of_cycle?
+          t('application_form.referees.info.cancelled_at_end_of_cycle')
+        elsif reference.cancelled?
+          t('application_form.referees.info.cancelled')
+        elsif reference.feedback_overdue?
+          t('application_form.referees.info.feedback_overdue')
+        elsif reference.feedback_requested?
+          t('application_form.referees.info.feedback_requested')
+        elsif reference.email_bounced?
+          t('application_form.referees.info.email_bounced')
+        end
+
+      return '' if text.blank?
+
+      if text.is_a?(Array)
+        text.inject('') { |prev, line| prev + tag.p(line, class: 'govuk-body govuk-!-margin-top-2') }.html_safe
```

Comment:
First thought - does returning an array of `tag.p` rather than a string have the same effect when rendering?

If not, this might be clearer:
 ` text.each_with_object('') { |line, content| content + tag.p(line, class: 'govuk-body govuk-!-margin-top-2') }.html_safe`

Reply:
I tried with an array of 'tags' but I doesn't render properly - I think because the return value from this method is already appended to another (html_safe) string. I couldn't help thinking there was some kind of `array_of_tags` psuedo-tag-helper type thing but didn't find it. So just changed to use `each_with_object`.